web/webkit/src/main/scala/net/liftweb/http/LiftServlet.scala

```diff
@@ -788,7 +789,7 @@ class LiftServlet extends Loggable {
 
       case ar: AnswerRender =>
         answers = ar :: answers
-        LAPinger.schedule(this, BreakOut(), 5.millis)
+        LAPinger.schedule(this, BreakOut(), 5)
```

Comment:
it expects a `Long` instead of a `TimeSpan` which is different than what `Scheduke.schedule` expects (may be a good candidate for a new PR to make them all use the same type)